src/js/hooks/cycleCount/useResolveStep.js

```diff
@@ -43,17 +43,41 @@ const useResolveStep = () => {
     currentLocation: state.session.currentLocation,
   }));
 
+  const mergeCycleCountItems = (items) => {
+    const duplicatedItems = _.groupBy(items,
+      (item) => `${item.binLocation?.id}-${item?.inventoryItem?.lotNumber}`);
+    return Object.values(duplicatedItems).map((itemsToMerge) => {
+      const maxCountIndex = _.maxBy(itemsToMerge, 'countIndex').countIndex;
```

Comment:
Perhaps this is worth moving to CycleCountDto? CycleCount already has `maxCountIndex` (this is not a change request, but a thought)